packages/web3-state/src/NameService.ts

```diff
@@ -56,7 +56,7 @@ export class NameServiceState<
         })
     }
 
-    async lookup(chainId: ChainId, name: string) {
+    lookup = async (chainId: ChainId, name: string) => {
```

Comment:
What's the difference with the previous version?

Reply:
the scope of `this`.